README.md

```diff
@@ -67,46 +67,103 @@ What is needed?
+  slot.
 
 #### `[[Builtin]]` internal slot
 
-All built-ins, with the exception of the intrinsic object `%ObjectPrototype%`,
-would have a `[[Builtin]]` internal slot with a string value identifying the
-name of the built-in. For instance, for the `%Math%` intrinsic object, the
-value of `[[Builtin]]` is `'Math'`.
-
-For `%TypedArray%` intrinsic objects, the value of `[[Builtin]]` is equal to
-the value of `[[TypedArrayName]]`. For instance, for a `%TypedArray%` with
-`[[TypedArrayName]]` `'Uint8Array'`, the value of `[[Builtin]]` is
-`'Uint8Array'`.
+Intrinsic objects listed in the table below have a `[[Builtin]]` internal slot
+with the given string value. Intrinsic objects not listed in the table do *not*
+have the `[[Builtin]]` internal slot.
+
+| Intrinsic Name        | Builtin Name          |
+| --------------------- | --------------------- |
+| `%Array%`             | `'Array'`             |
+| `%ArrayBuffer%`       | `'ArrayBuffer'`       |
+| `%AsyncFunction%`     | `'AsyncFunction'`     |
+| `%Atomics%`           | `'Atomics'`           |
+| `%Boolean%`           | `'Boolean'`           |
+| `%DataView%`          | `'DataView'`          |
+| `%Date%`              | `'Date'`              |
+| `%Error%`             | `'Error'`             |
+| `%EvalError%`         | `'EvalError'`         |
+| `%Float32Array%`      | `'Float32Array'`      |
+| `%Float64Array%`      | `'Float64Array'`      |
+| `%Function%`          | `'function'`          |
+| `%GeneratorFunction%` | `'GeneratorFunction'` |
+| `%Int8Array%`         | `'Int8Array'`         |
+| `%Int16Array%`        | `'Int16Array'`        |
+| `%Int32Array%`        | `'Int32Array'`        |
+| `%JSON%`              | `'JSON'`              |
+| `%Map%`               | `'Map'`               |
+| `%Math%`              | `'Math'`              |
+| `%Number%`            | `'Number'`            |
+| `%Object%`            | `'object'`            |
+| `%Promise%`           | `'Promise'`           |
+| `%RangeError%`        | `'RangeError'`        |
+| `%ReferenceError%`    | `'ReferenceError'`    |
+| `%Reflect%`           | `'Reflect'`           |
+| `%RegExp%`            | `'RegExp'`            |
+| `%Set%`               | `'Set'`               |
+| `%SharedArrayBuffer%` | `'SharedArrayBuffer'` |
+| `%String%`            | `'String'`            |
+| `%Symbol%`            | `'Symbol'`            |
+| `%SyntaxError%`       | `'SyntaxError'`       |
+| `%TypeError%`         | `'TypeError'`         |
+| `%Uint8Array%`        | `'Uint8Array'`        |
+| `%Uint8ClampedArray%` | `'Uint8ClampedArray'` |
+| `%Uint16Array%`       | `'Uint16Array'`       |
+| `%Uint32Array%`       | `'Uint32Array'`       |
+| `%URIError%`          | `'URIError'`          |
+| `%WeakMap%`           | `'WeakMap'`           |
+| `%WeakSet%`           | `'WeakSet'`           |
+
+*Note*: Currently, intrinsic prototype objects such as `%DatePrototype%`
```

Comment:
Ok, I'll need to go through and identify exactly which exotic objects this would include and add those to the list. I'll do that today then will get this PR landed.